.gitignore

```diff
@@ -6,6 +6,10 @@
 /pkg/
 /spec/reports/
 /tmp/
+Gemfile.lock
```

Comment:
> Chore: Remove Gemfile.lock from VCS. Although the Bundler team [now recommends on having it](https://bundler.io/guides/faq.html#using-gemfiles-inside-gems), that only makes sense with additional tooling in place (e.g. Dependabot). Without them, this is only making it harder to test against your dependencies and validate your gemspec properly.

Could you explain more context here? I think this is still useful to have in VCS to ensure all contributors are using the same set of dependencies, especially as it's recommended by bundler. In what ways are you finding it's harder to test against your dependencies and validate your gem spec?

Reply:
Following your suggestion, I'll push the `Gemfile.lock` back into VCS and open a separate issue about this 👍